x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_chart_embedded_map.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState, useRef, useEffect } from 'react';
+import uuid from 'uuid';
+import {
+  ErrorEmbeddable,
+  ViewMode,
+  isErrorEmbeddable,
+} from '../../../../../../../src/plugins/embeddable/public';
+import {
+  MapEmbeddable,
+  MapEmbeddableInput,
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
+} from '../../../../../maps/public/embeddable';
+import { useMlKibana } from '../../contexts/kibana';
+import { MAP_SAVED_OBJECT_TYPE } from '../../../../../maps/common/constants';
+import { LayerDescriptor } from '../../../../../maps/common/descriptor_types';
+import { getMLAnomaliesActualLayer, getMLAnomaliesTypicalLayer } from './map_config';
+// import { AnomalyRecordDoc } from '../../../../common/types/seriesConfig.chartData';
+
+interface Props {
+  seriesConfig?: any; // object;
+  severity: number;
+  tooltipService: any; // object;
+}
+
+export function EmbeddedMapComponent({ seriesConfig, severity, tooltipService }: Props) {
+  const [embeddable, setEmbeddable] = useState<MapEmbeddable | ErrorEmbeddable | undefined>();
+
+  const embeddableRoot: React.RefObject<HTMLDivElement> = useRef<HTMLDivElement>(null);
+  const layerList = useRef<LayerDescriptor[]>([]);
+  const {
+    services: { embeddable: embeddablePlugin, maps: mapsPlugin },
+  } = useMlKibana();
+
+  if (!embeddablePlugin) {
+    throw new Error('Embeddable start plugin not found');
+  }
+  if (!mapsPlugin) {
+    throw new Error('Maps start plugin not found');
+  }
+
```

Comment:
You're correct - will update to optional as it should be and show a toast with a message that the plugin is disabled but let the anomaly explorer load normally otherwise 👍